<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1218</width>
    <height>1087</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #f8fafc, stop:1 #e2e8f0);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frame_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>41</width>
         <height>80</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="pushButton_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background: transparent;
border: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imagem/icons/back.png</normaloff>../imagem/icons/back.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_6">
      <property name="minimumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>90</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>70</width>
         <height>90</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>1000</width>
         <height>1000</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../imagem/MOTORMAX-LGO.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>1000</width>
        <height>80</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>80</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>890</width>
         <height>70</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>890</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
font-size: 22px;
margin-top: 30px;
margin-bottom: 15px;
font: plain;
}</string>
       </property>
       <property name="text">
        <string>Cadastrar novo Cliente</string>
       </property>
      </widget>
     </widget>
    </item>
    <item alignment="Qt::AlignHCenter">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>1200</width>
        <height>720</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>1200</width>
        <height>720</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgba(255, 255, 255, 240);
border-radius: 15px;
border: 1px solid rgba(0, 0, 0,0.1); </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>420</x>
         <y>580</y>
         <width>150</width>
         <height>45</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #f59e0b;
color: black;
border: none;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: #d97706;
}

QPushButton:pressed{
background-color: #b45309;
}

QPushButton:disable{
background-color: #582503;
}</string>
       </property>
       <property name="text">
        <string>Salvar</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>75</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Nome *</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>620</x>
         <y>580</y>
         <width>150</width>
         <height>48</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: transparent;
color: Black;
border: 2px solid;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: rgb(234, 227, 174);
}

QPushButton:Pressed{
background-color: rgb(210, 203, 150);
}</string>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>100</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: Cleiton@gmail.com</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>75</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>CPF *</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>460</x>
         <y>75</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Email *</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>100</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: Cleiton</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>800</x>
         <y>100</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>1</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
       </property>
       <property name="inputMask">
        <string>999.999.999-99</string>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
       <property name="placeholderText">
        <string>Ex: 000.000.000-00</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>165</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Rockwell</family>
         <pointsize>12</pointsize>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;</string>
       </property>
       <property name="text">
        <string>Telefone *</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>190</y>
         <width>420</width>
         <height>45</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QComboBox {
border: 2px solid #e5e7eb;
border-radius: 8px;
padding: 12px;
font-size: 14px;
color: #374151;
}
QComboBox:hover{
background-color: rgb(234, 236, 240);
}

QComboBox:focus{
border: 2px solid #7f8082;
}

QComboBox::drop-down {
border: none;
}

QComboBox::down-arrow {
image: url(C:/Users/victor/Documents/vscode/python/curso/imagem/icons/arrow-down.png);
width: 18px;
height: 18px;
right: 6px;
}</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <property name="maxVisibleItems">
        <number>5</number>
       </property>
       <property name="insertPolicy">
        <enum>QComboBox::InsertAtCurrent</enum>
       </property>
       <property name="minimumContentsLength">
        <number>0</number>
       </property>
      </widget>
      <widget class="QFrame" name="frame_3">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>250</y>
         <width>1000</width>
         <height>90</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLineEdit" name="lineEdit_7">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>300</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>1</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit[echoMode=&quot;2&quot;], QLineEdit[echoMode=&quot;0&quot;] {
                border: 2px solid #e5e7eb;
                border-radius: 8px;
                padding: 12px;
                font-size: 14px;
                color: #374151;
            }
QLineEdit:focus{
border: 2px solid #7f8082;
}

QLineEdit:hover{
background-color: rgb(234, 236, 240);
}</string>
        </property>
        <property name="inputMask">
         <string>(99) 99999-9999</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
        <property name="placeholderText">
         <string>Ex: 9xxxx-xxxx</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>221</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Rockwell</family>
          <pointsize>12</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
background: transparent;</string>
        </property>
        <property name="text">
         <string>Novo Telefone *</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_5">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>10</y>
          <width>35</width>
          <height>48</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>35</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>35</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
background-color: transparent;
color: Black;
border: 2px solid;
border-radius: 8px;
padding: 12px;
font-size: 14px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: rgb(223, 223, 223);
}

QPushButton:Pressed{
background-color: rgb(195, 195, 195);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../imagem/icons/delete.png</normaloff>../imagem/icons/delete.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>970</x>
         <y>200</y>
         <width>141</width>
         <height>40</height>
        </rect>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #f59e0b;
color: black;
border: none;
border-radius: 8px;
font-size: 11px;
font-weight: bold;
min-height: 20px;
}

QPushButton:hover{
background-color: #d97706;
}

QPushButton:pressed{
background-color: #b45309;
}

QPushButton:disabled{
background-color: #582503;
}</string>
       </property>
       <property name="text">
        <string>Adicionar Telefone</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1218</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
